addons/mail/static/src/core/common/message_model.js

```diff
@@ -77,8 +77,8 @@ export class Message extends Record {
     isReadBySelf = Record.attr(false, {
         compute() {
             return (
-                this.thread.selfMember?.seen_message_id?.id >= this.id &&
-                this.thread.selfMember?.new_message_separator > this.id
+                this.thread?.selfMember?.seen_message_id?.id >= this.id &&
+                this.thread?.selfMember?.new_message_separator > this.id
```

Comment:
I thought all messages were inserted with `message_format`, thus they necessarily had `.thread` set?
IMO I don't mind being defensive on all non-id fields, conceptually when we just look at JS models this makes sense. But just want to make sure we can explain this change

Reply:
The whole file guards `this.thread` so I assume it is needed.  I remember I had a problematic flow but I can't remember which one.

Reply:
model and res_id are not required in python

Possibly inbox message unrelated to a thread.